ember-tasking/watchdog.c

```diff
@@ -174,5 +175,4 @@ void set_up_rtc_watchdog_final()
 void set_up_rtc_watchdog_1sec()
 {
     set_up_rtc_watchdog(FW_UPDATE_TIMEOUT_MS);
-}
-
+}
```

Comment:
Please fix missing newline at end of file

Reply:
> @ajaku We tested this on 1.1, correct? Please squash and merge!

Wasn't able to test on 1.1 last week because of some scons build issues on my laptop